packages/types/src/event.ts

```diff
@@ -46,6 +47,9 @@ export interface Event {
   debug_meta?: DebugMeta;
   // A place to stash data which is needed at some point in the SDK's event processing pipeline but which shouldn't get sent to Sentry
   sdkProcessingMetadata?: { [key: string]: any };
+  transaction_info?: {
+    source: TransactionSource;
+  };
```

Comment:
Tiny nit or question:
Okay, so `transaction_info` is optional but `.source` is mandatory. According to the spec, both are recommended. Do we translate the spec here correctly or should we make both mandatory/optional?

Reply:
For our case in the SDK, if someone sets `transaction_info`, they should always be setting a `source`, as it makes no sense to send an empty `transaction_info` object. Let's keep it this way (disjoint from the spec) for now, we can come back and iterate if needed.